aspnetcore/signalr/messagepackhubprotocol.md

```diff
@@ -5,23 +5,189 @@ description: Add MessagePack Hub Protocol to ASP.NET Core SignalR.
 monikerRange: '>= aspnetcore-2.1'
 ms.author: bradyg
 ms.custom: mvc
-ms.date: 11/12/2019
+ms.date: 04/13/2020
 no-loc: [SignalR]
 uid: signalr/messagepackhubprotocol
 ---
+
 # Use MessagePack Hub Protocol in SignalR for ASP.NET Core
 
-By [Brennan Conroy](https://github.com/BrennanConroy)
+::: moniker range=">= aspnetcore-5.0"
+
+This article assumes the reader is familiar with the topics covered in [Get Started](xref:tutorials/signalr).
+
+## What is MessagePack?
+
+[MessagePack](https://msgpack.org/index.html) is a fast and compact binary serialization format. It's useful when performance and bandwidth are a concern because it creates smaller messages compared to [JSON](https://www.json.org/). The binary messages are unreadable when looking at network traces and logs unless the bytes are passed through a MessagePack parser. SignalR has built-in support for the MessagePack format and provides APIs for the client and server to use.
```

Comment:
Should "What is MessagePack?" be inside a moniker-specific section?

Reply:
@scottaddie ?

Reply:
It's already inside of a section. See line 15.